common/djangoapps/student/models.py

```diff
@@ -1002,7 +1005,6 @@ def __unicode__(self):
         ).format(self.user, self.course_id, self.created, self.is_active)
 
     @classmethod
-    @transaction.atomic
```

Comment:
@rlucioni I have reverted retry logic, and leave it only with `default keywords`. Please review.